### ISSUE TITLE
[Feedback] Shared cockpit and VATSIM

### ISSUE THREAD

**Issue by MoritzT** [Sequal32/yourcontrol#16]
Hey Sequal32,

JonFly and I tested the addon you wrote and overall we really enjoy it. We just did a test run again, but this time using VATSIM. Me controlling the aircraft and Jon doing coms. (https://www.twitch.tv/videos/747079454)
BAT and PITOT switches were not synced, but after using the latest version aka modifying the dat file it worked fine.

Jon was even able to bring up gears without me handing him controls which is awesome.

We noticed though that squawk did not work properly. code was synced, but when I switched to ALT reporting in the G1000 it did not sync that.
It also appears that radio standby/active frequencies are not correctly synced and we both had to make sure to set the right standby frequency before I switched them.

Transfer of controls is still wonky and messes things up, so we stopped using that for now.

This is an awesome addon and we really enjoy using it.

**Comment by Sequal32** [comment]
Thanks a bunch for the feedback! There are a bunch of switches/avionics in the aircraft that are not exposed to SimConnect, but I'm currently seeking to find a way to bridge that gap for next release. Can you go a little more in depth to why transferring controls is wonky?

**Comment by MoritzT** [comment]
So I am in NY and Jon is in SLC when we transfer controls we almost always get the buffer overload issue. We tried different update rates and buffer sizes.

First time we tried Jon's plane jumped ahead of mine and our positions were out of sync and the auto pilot was broken afterwards (172 steam gauge) 

Second time we had buffer overload and then just didn't try again.

During takeoff my plane was a like 2 lengths ahead of Jon as well which is unrelated to control switch, but maybe due to update rate. Controls were still in sync, but I had to do a lot get landing to make sure my tow gets on the runway.
